plugins/ship-mate/skills/scan/SKILL.md

```
@@ -0,0 +1,218 @@
+context-mode         →  /plugin list | grep context-mode
+```
+
+These plugins are **optional**. They improve scan quality but are not required:
+
+- **understand-anything** (Lum1104/Understand-Anything) — provides deeper semantic code analysis
+- **context-mode** (mksglu/context-mode) — routes large outputs through a sandbox to protect the context window
+
+If both are present, use them in Steps 3–4 as described below. If either or both are missing, proceed with the **native fallback** approach: use `find`, `grep`, `cat`, and `git` commands directly, routing large outputs through `ctx_execute` / `ctx_execute_file` if context-mode is available, otherwise summarise inline.
+
+> **Note:** To install the optional plugins manually:
+> ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)